client-rpc/src/wallet_rpc.rs

```diff
@@ -6,16 +6,27 @@ use secstr::SecUtf8;
 use serde::{Deserialize, Serialize};
 
 use chain_core::common::{H256, HASH_SIZE_256};
+
 use chain_core::init::coin::Coin;
 use chain_core::tx::data::address::ExtendedAddr;
 use chain_core::tx::data::attribute::TxAttributes;
 use chain_core::tx::data::output::TxOut;
-use client_common::balance::TransactionChange;
+use client_common::balance::BalanceChange;
 use client_common::{Error, ErrorKind, PublicKey, Result as CommonResult};
 use client_core::{MultiSigWalletClient, WalletClient};
 
 use crate::server::{rpc_error_from_string, to_rpc_error};
 
+#[derive(Serialize, Deserialize)]
+pub struct RowTx {
+    kind: String,
+    transaction_id: String,
+    address: String,
+    height: String,
```

Comment:
height is `i64` from Tendermint's protobufs and it'll be "0, 1, 2, 3 ...", so I guess it'd be parse-friendly for a long time

Reply:
yes, javascript handles all i64 or u64 in strings.